### PR TITLE
fix the bug that 'username' in broker load is invalid 

### PR DESCRIPTION
**Pull request by frwrdt**
fix bug in issue #3234

### CODE REVIEW

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java [hunk 1]
```diff
@@ -287,9 +285,6 @@ public BrokerFileSystem getDistributedFileSystem(String path, Map<String, String
                                     e.getMessage());
                         }
                     }
-                } else {
-                    throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
-                            "invalid authentication.");
```
Comment (HangyuanLiu) [inline fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java:292]: We need to determine whether the authentication method exists

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java [hunk 2]
```diff
@@ -338,7 +333,20 @@ public BrokerFileSystem getDistributedFileSystem(String path, Map<String, String
                 }
 
                 conf.set(FS_HDFS_IMPL_DISABLE_CACHE, "true");
-                FileSystem dfsFileSystem = FileSystem.get(pathUri.getUri(), conf);
+                FileSystem dfsFileSystem = null;
```
Comment (HangyuanLiu) [inline fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java:336]: Is this method compatible with  versions of hdfs?

**Comment by HangyuanLiu** [comment]
Please change to a understandable title

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java (hunk 1 above)
Comment (frwrdt) [reply fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java:292]: The authentication has been checked before, so this is  redundant here.

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java (hunk 2 above)
Comment (frwrdt) [reply fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java:336]: The version 2.7.3 of package _hadoop-common_ including this method is the same as package _hadoop-hdfs_.

**Review by HangyuanLiu — APPROVED** [review]

**Pull request merged**